File: 프로그래머스/lv1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):    
    answer = []
    for i in range(len(numbers)):
        for j in range(i + 1, len(numbers)):
            if numbers[i] + numbers[j] in answer:
                continue
            answer.append(numbers[i] + numbers[j])
    answer.sort()
    return answer

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.*;

class Solution {
    public int[] solution(int[] numbers) {
        Set<Integer> set = new HashSet<>();
        for(int i = 0; i < numbers.length; i++){
            for(int j = i + 1; j < numbers.length; j++){
                set.add(numbers[i] + numbers[j]);
            }   
        }
                
        int[] answer = new int[set.size()];
        int idx = 0;
        for(int n: set){
            answer[idx] = n;
            idx++;
        }
        Arrays.sort(answer);
        return answer;
    }
}